docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -3,31 +3,42 @@
 
 **Tip**: If you are new to AutoGluon, review :ref:`sec_tabularquick` to learn the basics of the AutoGluon API.
 
-This tutorial describes how you can exert greater control when using AutoGluon's `fit()` by specifying the appropriate arguments. Using the same census data table as :ref:`sec_tabularquick`, we will try to predict the `occupation` of an individual - a multi-class classification problem.
+This tutorial describes how you can exert greater control when using AutoGluon's `fit()` or `predict()`. Recall that to maximize predictive performance, you should always first try `fit()` with all default arguments except `eval_metric` and `presets`, before you experiment with other arguments covered in this in-depth tutorial like `hyperparameter_tune`, `hyperparameters`, `stack_ensemble_levels`, `num_bagging_folds`, `num_bagging_sets`, etc.
 
-Start by importing AutoGluon, specifying TabularPrediction as the task, and loading the data.
+Using the same census data table as before, we'll now predict the `occupation` of an individual - a multiclass classification problem. Start by importing AutoGluon, specifying TabularPrediction as the task, and loading the data.
 
 ```{.python .input}
 import autogluon as ag
 from autogluon import TabularPrediction as task
 
+import numpy as np
+
 train_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
-train_data = train_data.head(500) # subsample 500 data points for faster demo (comment this out to run on full dataset instead)
+subsample_size = 500 # subsample subset of data for faster demo, try setting this to much larger values
```

Comment:
small nit: `subsample_size = 500  #` (two spaces before comment instead of one)

This applies to all instances of this. Most python linters will highlight this as a PEP8 violation when only 1 space is used instead of 2 before an in-line comment. Adding the 2nd space will make the tutorial look cleaner for users where this linting is enabled.